automol/util/dict_/_dict_.py

```diff
@@ -100,7 +101,7 @@ def value_in_floatkey_dct(dct: dict[object, object], key: object, tol=1.0e-5):
     decimal places or strings.
```

Comment:
The return types for `value_by_unordered_key` and `value_in_floatkey_dct` should both be `-> object`.

Reply:
Most of the functions below have missing return types as well. In order, they are:
```
values_in_multilevel_dct(...) -> object:
keys_by_value(...) -> frozenset[object]:
transform_keys(...) -> dict[object, object]:
```